jhipster-server/pom.xml

```diff
@@ -213,17 +213,6 @@
 
     <build>
         <plugins>
-            <plugin>
-                <groupId>org.sonatype.plugins</groupId>
```

Comment:
are you sure this is not needed anymore? I think I need this to publish snapshot releases

Reply:
The same config is in the parent:
https://github.com/jhipster/jhipster/blob/master/jhipster-dependencies/pom.xml#L839
And we just need to enable the plugin:
https://github.com/jhipster/jhipster/pull/55/files#diff-580af11172ef8dd3e17b50bdf019f094L250